packages/components/src/passengerCard/__tests__/__snapshots__/PassengerCard.test.js.snap

```diff
@@ -146,7 +146,7 @@ Object {
         <PassengerCardDetail
           label="Date of birth"
           style="normal"
-          value="22/04/1980"
+          value="1980-04-12"
```

Comment:
Is this change related to format used in MMB? I think that so far we are using `DD/MM/YYYY` everywhere and it will be reasonable to keep it unified.

Reply:
@JoseDuSV Sure! I just didn't recall which format was in the config 😆So I put the first one